src/components/AttachmentCarousel/index.js

```diff
@@ -211,24 +230,29 @@ class AttachmentCarousel extends React.Component {
                                 iconFill={themeColors.text}
                                 iconStyles={[styles.mr0]}
                                 onPress={() => this.cycleThroughAttachments(-1)}
-                                isDisabled={this.state.isForwardDisabled}
                             />
                         )}
                     </>
                 )}
-                <CarouselActions
-                    styles={[styles.attachmentModalArrowsContainer]}
-                    canSwipeLeft={!this.state.isBackDisabled}
-                    canSwipeRight={!this.state.isForwardDisabled}
-                    onPress={() => this.canUseTouchScreen && this.toggleArrowsVisibility(!this.state.shouldShowArrow)}
-                    onCycleThroughAttachments={this.cycleThroughAttachments}
-                >
-                    <AttachmentView
-                        onPress={() => this.toggleArrowsVisibility(!this.state.shouldShowArrow)}
-                        source={authSource}
-                        file={this.state.file}
+                {this.state.isLoading ? (
+                    <FullScreenLoadingIndicator
+                        style={[styles.opacity1, styles.bgTransparent]}
                     />
```

Comment:
Not sure we need a full screen loading, may be we should check with design

cc: @chiragsalian 

Either way loading should not scrollable

https://user-images.githubusercontent.com/85645967/221643461-4f768079-b0ab-4a80-a09b-85b40387525a.mov